invenio_records_rest/errors.py

```diff
@@ -177,3 +177,14 @@ def __init__(self, options=None, **kwargs):
         super(RESTException, self).__init__(**kwargs)
         self.description = 'No completions requested.{0}'.format(
             ' (options: {0})'.format(options) if options else '')
+
+
+class JSONSchemaValidationError(RESTValidationError):
+    """JSONSchema validation error exception."""
+
+    code = 400
+
+    def __init__(self, description=None, error=None, **kwargs):
```

Comment:
Do you really need the description here? It feels a bit odd 😉 

Also something like this might be more reliable

```
self.description = 'Validation error: {0}.'.format(error.message if error else '')
```